src/addIndex.js

```diff
@@ -38,6 +38,6 @@ module.exports = _curry1(function addIndex(fn) {
       return result;
     };
 
-    return fn.apply(this, _prepend(indexedFn, _slice(arguments, 1)));
+    return fn.apply(this, prepend(indexedFn, _slice(arguments, 1)));
```

Comment:
Meh, I'm still not a big fan of the `addIndex` functions (too hacky and can't be extended for `addKey`). I'd still rather see the `Index`/`Key` functions be the default and the none `key`/`index` functions just apply `unary` on the predicate